README.md

```diff
@@ -205,4 +205,6 @@ Blender: Original by Benjamin Amos
 
 Fried Chicken: https://pixabay.com/en/chicken-fried-leg-piece-poultry-37807/
 
-Chicken: http://www.clipartpanda.com/clipart_images/shared-by-leonnardo00-9075011
\ No newline at end of file
+Chicken: http://www.clipartpanda.com/clipart_images/shared-by-leonnardo00-9075011
+
+Toast: http://piq.codeus.net/picture/55650/toast (edited)(resized)
```

Comment:
That toast looks fine, so long as there is a page we can link that explains the image and its licensing :-)